src/lib/services/state-service.ts

```diff
@@ -633,7 +633,7 @@ export default class StateService {
     async importTagTypes(
         tagTypes: ITagType[],
         keepExisting: boolean,
-        oldTagTypes: ITagType[] = [], // eslint-disable-line
```

Comment:
I believe same comment as in the other parameter with a default value and // eslint-disable-line

Reply:
Oh, I'm very intrigued now... this was added by renovate bot! https://github.com/Unleash/unleash/pull/1147/files#diff-8757be4693afd352b1e0507343d2de88066e2d8ec2f453a11be93920b9d0b182L599

Reply:
That's very odd.